drivers/clock_control/Kconfig.stm32

```diff
@@ -8,7 +8,7 @@ menuconfig CLOCK_CONTROL_STM32_CUBE
 	bool "STM32 Reset & Clock Control"
 	depends on SOC_FAMILY_STM32
 	select USE_STM32_LL_UTILS
-	select USE_STM32_LL_RCC if SOC_SERIES_STM32MP1X
+	select USE_STM32_LL_RCC if SOC_SERIES_STM32MP1X || SOC_SERIES_STM32H7X
```

Comment:
Miss (for H7)
         select USE_STM32_LL_BUS if SOC_SERIES_STM32H7X
         select USE_STM32_LL_PWR if SOC_SERIES_STM32H7X
         select USE_STM32_LL_SYSTEM if SOC_SERIES_STM32H7X

Reply:
Updated, thanks for noting. Regarding `USE_STM32_LL_PWR` I can't see any reference to `LL_PWR_*` on the H7 clock control driver, so I haven't added it.